pkg/cmd/serve/cmd.go

```diff
@@ -20,6 +22,11 @@ func NewServeCommand(env *environments.Env) *cobra.Command {
 			if err != nil {
 				glog.Fatalf("Unable to initialize environment: %s", err.Error())
 			}
+			info, e := buildinformation.GetBuildInfo()
+			if e != nil {
+				glog.Fatal("unable to retrieve buildinfo")
```

Comment:
Here the error contenct can be printed, similarly to what's done in the unable to initialize environment error logging